docs/pages/admin-guides/access-controls/sso/gitlab.mdx

```diff
@@ -71,7 +71,7 @@ $ tctl sso configure oidc --preset gitlab \
 --id <APPLICATION-ID> \
 --secret $( cat client-secret.txt) \
 --claims-to-roles groups,company/admin,admin \
---claims-to-roles groups,company/dev,dev > oidc.yaml
+--claims-to-roles groups,company/access,access > oidc.yaml
```

Comment:
this should still be `dev` right?
```suggestion
--claims-to-roles groups,company/dev,access > oidc.yaml
```

Same comment for line 19 as well, as we are only updating `dev -> access` in Teleport but keeping the `dev` reference as it is from what is coming from the IdP.